setup.py

```diff
@@ -17,7 +17,7 @@
     license='AGPL',
     url='http://github.com/edx/edx-django-release-util',
     install_requires=[
-        'django>=1.8,<1.11',
+        'django>=1.8,<2.0',
```

Comment:
If we do this, and Django decided to have django 1.12, we will be suffering from breakage.
Are you sure Django won't have version 1.12? Why not update this to be `<1.12`?